setup.py

```diff
@@ -140,6 +145,11 @@ def read(*names, **kwargs):
         'extractcode-7z',
 
         # commoncode
+        # Hack to avoid broken backports.os on systems with old configparser
+        # See https://github.com/PiDelport/backports.os/issues/11
+        # and https://github.com/jaraco/configparser/issues/17
+        'configparser >= 0.6.0',
```

Comment:
I see the min version of configparser is 3.2

Reply:
Yes, that should be something like 3.5.1, not 0.6.0.